### PR TITLE
Block Interaction is now Synchronized with VanillaCamera 

### PR DESCRIPTION
**Pull request by Waterpicker**
Added PlayerInteract and replace InteractComponent with it.

Waterpicker Waterpicker@rocketmail.com

### CODE REVIEW

File: src/main/java/org/spout/vanilla/component/entity/player/PlayerInteract.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License,
+ * the MIT license and the Spout License Version 1 along with this program.
+ * If not, see <http://www.gnu.org/licenses/> for the GNU Lesser General Public
+ * License and see <http://spout.in/licensev1> for the full license, including
+ * the MIT license.
+ */
+/*
+ * To change this template, choose Tools | Templates
```
Comment (Zidane) [inline src/main/java/org/spout/vanilla/component/entity/player/PlayerInteract.java]: Remove this for me :P

Comment (Waterpicker) [inline src/main/java/org/spout/vanilla/component/entity/player/PlayerInteract.java]: Tell me how to do this via git bash and gladly.

Comment (windy1) [inline src/main/java/org/spout/vanilla/component/entity/player/PlayerInteract.java]: - Make the changes
- Do `git add . -A`
- Do `git commit --amend`
- Exit the commit editor
- Do `git push --force origin master`

**Comment by Waterpicker** [comment]
@Zidane Fixed

**Pull request merged**